static/stories/_publication/slides-publication.md

```diff
@@ -0,0 +1,161 @@
+Reusability - it's only reusable if it's available, and if people know what they are allowed to do with it (based on its software license)
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## Traditional publishing pipeline
+
+* Write some code and leave it on your harddrive
+* Generate scientific results with it
+* Write a paper about those results
+* Possibly not even mentioning the software at all
+
+### Consequences
+
+* The software itself is not citable
+* Methods are not easily reproduced or even analysed
+* Nobody else can build on top of your research
+
+---
+
+<!-- .slide: data-state="standard" -->
+## Modern publishing pipeline
+
+* Software repository
+* Tag/semantic versioning
+* Get a persistent identifier
+* Registries
+* Software Paper
+
+---
+
+<!-- .slide: data-state="standard" -->
+### Software repository
+
+Put all source files in a public version-controlled repository
+e.g. github, gitlab
+
+* Software is immediately Findable
+* Naturally contains a log of all development history
+
+Include a software license, and a citation.cff file
+
+Note:
+
+See license, citation and software management chapters
+
+---
+
+<!-- .slide: data-state="standard" -->
+### Software repository
+Which one?
+<img src="stories/_publication/repository_logos.png" width="80%">
+
+Note:
+
+Anything is better than nothing. Gitlab and Github are the mainstream, used by many and with lots of features. You might run into projects using other repositories. 
+
+---
+
+<!-- .slide: data-state="standard" -->
+### Tag/semantic versioning
+
+It is important to communicate the actual version of the software used for a particular piece of research
+
+If your software is used as a dependency of another software/workflow then specific versions can be referenced
+
+---
+
+<!-- .slide: data-state="standard" -->
+### Get a Persistent Identifier
+
+Specific versions of the software should be published to an archiving service that provides persistent identifiers
+
+For example, Zenodo is a long term archiving service for scientific data and software.
+Zenodo will assign a DOI to the published data, making it citable
+
+Note:
+
+See Citation story
+
+---
+
+<!-- .slide: data-state="standard" -->
+### Registries
+Add a link to your software on a relevant public registry
+For example, the eScience Center maintains the Research Software Directory
+Can also be domain specific software registries
+Or even list it on awesomelist
+
+This aids greatly in Findability by relevant researchers
+
+---
+
+<!-- .slide: data-state="standard" -->
+### Software Paper
+
+Publish a journal article about the software specifically
+
+* Journal of Research Software (JORS)
+* Journal of Open Source Software (JOSS)
+* SoftwareX
+* CRAN
+
+Some journals perform peer review of the software in question
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## Automate as many steps as possible!
+
+- Demo
```

Comment:
I'm not sure whether we want to do a demo of the automating necessarily... We probably don't want to get into github actions at all with them, but just explain. Yet maybe we do, if it is easy enough

Reply:
I've added the demo steps to the Automated Software [...] tab and referred to it in the slides.